### PR TITLE
feat: Added Meetup History 

### LINKED ISSUE #548: Meetup History Backend
## Description
As a user,
I should be able to see the meetup history for each meetup,
so that I can get a rough idea about the meetup .

## Mocks
In progress

## Acceptance Criteria
When you click on meetup, you’ll get meetup history that shows resources.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3 days.

### PR DESCRIPTION
**Pull request by ritwickraj78**
### Description
 - Added Images Model and Resource Fields to Add Meetup and Edit Meetup Form
- Created past meetup form where user can upload resources and images after meetup is held.
- Created Past Meetup View to add the resources.

Fixes #548 

### Type of Change:

- Code

**Code/Quality Assurance Only**

- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

- All unit tests pass
- Flake test passes
- Travis Check successful.
- Tested locally

Screenshot from 2020-07-10 11-49-38
Screenshot from 2020-07-10 11-49-35
Screenshot from 2020-07-10 11-45-49
Screenshot from 2020-07-10 11-45-23
Screenshot from 2020-07-10 11-45-02

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by sakshi1499 — CHANGES_REQUESTED** [review]
Just a quick view, I think we have reviewed Pr on this once, did you close that one? Also, You haven't removed not coming yet.

**Comment by ritwickraj78** [comment]
> Just a quick view, I think we have reviewed Pr on this once, did you close that one? Also, You haven't removed not coming yet.

I had closed the eariler one..could you'll please review this one. Then I could make the changes

**Comment by ritwickraj78** [comment]
@SanketDG @sakshi1499 @sammy1997 Can you review this once? It needs to be merged

**Comment by SanketDG** [comment]
This is mostly reviewed from my side. I wanted a fresh pair of eyes to review this :)

File: systers_portal/static/css/style.css
Comment (sammy1997) [inline systers_portal/static/css/style.css]: @ritwickraj78 why are there CSS and JS changes on this commit if you have a separate commit for frontend?

File: systers_portal/meetup/forms.py
Comment (sammy1997) [inline systers_portal/meetup/forms.py:279]: @ritwickraj78 Should be PastMeetupForm. Also, as we already have an EditMeetupForm where we can edit the image field what is the need for this form?

**Comment by sammy1997** [comment]
@ritwickraj78 can you rebase and resolve the merge conflicts

**Comment by ritwickraj78** [comment]
@SanketDG Could you have a check on the migration files?

**Comment by SanketDG** [comment]
@ritwickraj78 `django-multiupload` does not have an official release for >= 2.2
https://github.com/Chive/django-multiupload/issues/31

We have to use the git version like: 

```
git+https://github.com/Chive/django-multiupload.git#egg=multiupload
```

I have opened an issue https://github.com/Chive/django-multiupload/issues/42

@sammy1997 @sakshi1499 @ritwickraj78 If we want it this way, I will push the changes accordingly.

Please note that this was largely part of the migration work, and that is why I am doing it, because it does not involve any of @ritwickraj78's work.

**Comment by ritwickraj78** [comment]
@SanketDG Changes done. Can you please create the fresh migrations?

File: requirements/prod.txt
Comment (SanketDG) [inline requirements/prod.txt:15]: We need to use the github version because that has support for Django >= 2.2

We can do two things here:
-  Consider this is done (not merge) and wait for the [release on PyPI](https://github.com/Chive/django-multiupload/issues/42)
- Merge now, open an issue to fix this later when it's released.

**Review by SanketDG — APPROVED** [review]
LGTM from my side except for that piece of code :slightly_frowning_face: 

Willing to hear what the other mentors are thinking

File: requirements/prod.txt
Comment (sakshi1499) [reply requirements/prod.txt:15]: I'd go with the second option honestly, I think its better if we merge it for now and not keep this PR open. What do you prefer more?

Comment (SanketDG) [reply requirements/prod.txt:15]: I also prefer the second option because I do not know when the package will be released or whether it will be released at all.

**Review by SanketDG — APPROVED** [review]

**Review by SanketDG — APPROVED** [review]
I am merging this!

**Review by sakshi1499 — CHANGES_REQUESTED** [review]
Not coming has still not been removed :/ @SanketDG

**Review by sakshi1499 — APPROVED** [review]
Changes added in other PR. We can merge this then!

**Pull request merged**
